spec/controllers/stories_bulk_update_controller_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+describe StoriesBulkUpdateController do
+  describe '#create' do
+    def catch_column(model, arg)
+      model.map do |record|
+        record.reload[arg.to_sym]
+      end
+    end
+
+    def response_body(status)
+      JSON.parse(response.body)[status]
+    end
+
+    before do
+      sign_in user_1
+    end
+
+    let(:user_1) { create(:user, :with_team) }
+    let(:user_2) { create(:user, :with_team) }
+    let(:user_3) { create(:user, :with_team) }
+    let(:user_4) { create(:user, :with_team) }
+    let(:project) { create(:project, users: [user_1, user_2, user_3], teams: [user_1.teams.first]) }
+    let(:model) { create_list(:story, 2, project: project, requested_by: user_1) }
```

Comment:
Why not call it `stories`? `model` doesn't say what's inside the variable.

Reply:
Really is better. Thanks